SpokestackTests/TFLiteWakewordRecognizerTest.swift

```diff
@@ -195,6 +207,12 @@ class TFLiteWakewordRecognizerTestDelegate: PipelineDelegate, SpeechEventListene
     
     func didDeactivate() {
         self.deactivated = true
+        guard let _ = self.didDeactivateExpectation else {
```

Comment:
or `if self.didDeactivateExpectation == nil {`

Reply:
i believe what is there is considered idiomatic